app/routes/events/view/speakers/list.js

```diff
@@ -76,6 +82,30 @@ export default class extends Route.extend(EmberTableRouteMixin) {
           }
         }
       ];
+    } else if (params.speakers_status === 'withdrawn') {
+      filterOptions = [
+        {
+          name : 'sessions',
+          op   : 'any',
+          val  : {
+            name : 'state',
+            op   : 'eq',
+            val  : 'withdrawn'
+          }
+        }
+      ];
+    } else if (params.speakers_status === 'canceled') {
+      filterOptions = [
```

Comment:
I think these conditions can be replaced by  - 
```
if (params.speakers_status) {
      filterOptions = [
           {
          name : 'sessions',
          op   : 'any',
          val  : {
            name : 'state',
            op   : 'eq',
            val  : params.speakers_status
          }
        }
      ];
    }
```

Reply:
what about "all" ?

Reply:
take that in else

Reply:
Check now please, thanks